fieldservice_route/models/fsm_route_dayroute.py

```diff
@@ -47,6 +58,9 @@ def _compute_order_count(self):
                                 string='Orders')
     order_count = fields.Integer(string='Number of Orders',
                                  compute=_compute_order_count)
+    order_max = fields.Integer(
+        related='route_id.max_order', string="Capacity",
```

Comment:
It would be best to be consistent and use the same filed name as in the original related model.